### PR TITLE
update metrics store feature page

### PR DESCRIPTION
**Pull request by sradco**
I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md):@sradco
This pull request needs review by: @sandrobonazzola

### CODE REVIEW

**Comment by mykaul** [comment]
@JohnMarksRH - can you review?

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 1]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+**oVirt Hypervisors and engine setup -**
+
+On upgrade from 4.1.0, oVirt machines already include Fluentd and Collectd packages.
+Now we need to deploy and configure Collectd and Fluentd to send the data to the central Metrics store:
+
+1. Install / Upgrade and setup oVirt Engine 4.1 latest.
+
+2. Install / Upgrade and activate one or more hosts, 4.1 latest.
+
+3. Copy the CA certificate, created in step 1 on the metrics store machine, to the engine machine.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Suggested rewrite
"Copy the CA certificate created earlier on the metrics store machine, to the engine machine."

Comment
I looked at the earlier step and the two links under "oVirt Metrics Store Setup". It's hard to identify the step described above and I don't see a reference to "CA certificate"

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 2]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+   Run on the metrics store machine:
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Suggest: "On the metrics store machine, run:"

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 3]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+        # scp /path/to/ca/certificate.cert root@<fqdn/of/engine/machine>:/etc/ovirt-engine-metrics
+
+4. Copy  /etc/ovirt-engine-metrics/config.yml.example  to config.yml.
+
+   Run on the oVirt engine machine:
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Suggest:

"On the oVirt engine machine, run:"

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 4]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+        # cp /etc/ovirt-engine-metrics/config.yml.example /etc/ovirt-engine/config.yml
+
+5. Update on the engine machine the file /etc/ovirt-engine-metrics/config.yml with the following lines:
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Suggest:
"Update the file /etc/ovirt-engine-metrics/config.yml located on the engine machine, with the following lines:

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 5]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+        # vi /etc/ovirt-engine-metrics/config.yml.example
+
+    Replace:
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Replace with what?

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 6]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+     
+     "fluentd-server.example.com"  -  The fully qualified domain name of the metrics store machine
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Suggest:

- "fluentd-server.example.com"  -  The fully qualified domain name of the metrics store machine

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 7]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+     
+     "my_shared_key" - The shared key configured in Fluentd on the metrics store machine and
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Suggest: 
 - "my_shared_key" - The shared key configured in Fluentd on the metrics store machine

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 8]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+     
+     "/path/to/fluentd_ca_cert.pem" - The path to the fluentd ca certificate.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Suggest:

-  "/path/to/fluentd_ca_cert.pem" - The path to the Fluentd CA certificate

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 9]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+6. Run on the engine machine, as root:
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Suggest:
"On the engine machine, run as root:"

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 10]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+        # /usr/share/ovirt-engine-metrics/setup/ansible/configure_ovirt_hosts_for_metrics.sh
+
+It runs the Ansible script that configures Collectd and Fluentd on the oVirt engine and hypervisors.
+
+It should finish without errors.
+
+When finished, you should be able to see statistics in the Kibana console, at the address you configured in step 1,  about your hosts, VMs, etc.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Step 1?

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 11]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
 
 *   Target Release: 4.1
 *   Status: Released
-*   Last updated: Mon 24 Jan 2017
+*   Last updated: Mon 12 Jun 2017
 
 ## Description
 
-[![oVirt Metrics Store Architecture](/images/wiki/MetricsStoreArchitecture.jpg)](images/wiki/MetricsStoreArchitecture.jpg)
+In 4.1, Collectd and Fluentd are already installed on the hosts and engine machines.
```
Comment (sandrobonazzola) [inline source/develop/release-management/features/engine/metrics-store.html.md]: collectd should be all lowercase (http://collectd.org), fluentd as well, all lowercase (http://www.fluentd.org/)

Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: updated

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 12]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+We have an Ansible script that configures them on all relevant hosts.
 
-In 4.1, Collectd will be already installed and configured on the hosts.
-We plan to further integrate it with external tools to collect, aggregate and visualize metrics and logs.
+We plan to continue adding additional logs and metrics as required, add pre-defined dashboards and aletring that will detect and notify the user in case something goes wrong in his environment, based on SLA events, thresholds etc.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 13]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
 
-In the next version, Collectd will be installed and configured with the engine as well.
+### oVirt Metrics Installation Instructions
 
-We also plan on adding Aletring that will detect and notify the user in case something goes wrong in his environment,
-based on SLA events, thresholds etc.
+**oVirt Metrics Store setup -**
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 14]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
 
-## See Also
+**Note:** Currently it should be installed on a new machine, separate from the engine. 
+It can be installed on a dedicated vm. 
 
-[oVirt Metrics – EFK](https://sradcoblog.wordpress.com/2016/07/19/ovirt-metrics-elk/)
+Please follow one of these options, adding SSO in optional:
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 15]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+ 
+  * [Metrics Store setup on Top of OpenShift with oVirt Engine SSO](https://www.ovirt.org/blog/2017/05/openshift-openId-integration-with-engine-sso/)
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 16]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+  OR
+
+  * [Metrics Store setup on Top of OpenShift without oVirt Engine SSO](https://github.com/ViaQ/Main/blob/master/README-mux.md)
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 17]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+[![oVirt Metrics Data Flow](/images/wiki/oVirtMetricsDataFlow.jpg)](images/wiki/oVirtMetricsDataFlow.jpg)
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 18]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+Once you finished this step, you should have:
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 19]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+  * Kibana - <https://kibana.{hostname}>
+  * Openshit portal - <https://openshift.{hostname}>
+
+
+**oVirt Hypervisors and engine setup -**
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 20]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+On upgrade from 4.1.0, oVirt machines already include Fluentd and Collectd packages.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 21]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+Now we need to deploy and configure Collectd and Fluentd to send the data to the central Metrics store:
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 22]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+1. Install / Upgrade and setup oVirt Engine 4.1 latest.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: latest version

Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: updated to "latest release"

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 23]
```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+2. Install / Upgrade and activate one or more hosts, 4.1 latest.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: updated to "latest release"

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 1 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: I added a link to the exact step where it was created please review again.
@JohnMarksRH

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 2 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 3 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 4 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 5 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Updated to "Replace the example values with your specific environment details:"

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 6 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 7 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 9 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 10 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: updated. Please review again @JohnMarksRH

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 24]
```diff
@@ -23,20 +23,81 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+6. On the engine machine, run as root:
+
+        # /usr/share/ovirt-engine-metrics/setup/ansible/configure_ovirt_hosts_for_metrics.sh
+
+It runs the Ansible script that configures Collectd and Fluentd on the oVirt engine and hypervisors.
+
+It should finish without errors.
+
+When finished, you should be able to see statistics in the Kibana console, at the address you configured previously on the [metrics store machine](https://github.com/ViaQ/Main/blob/master/README-mux.md#running-kibana), about your hosts, VMs, etc.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Suggested change:
Once finished, you can view host, VM and other statistics in the Kibhana console, at the address configured earlier on in this procedure (see metrics store machine). 

Comment: "metrics store machine" is the link.

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 25]
```diff
@@ -23,20 +23,81 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+Once you have finished this step, you should have:
+
+  * Kibana - <https://kibana.{hostname}>
+  * Openshit portal - <https://openshift.{hostname}>
+
+
+**oVirt Hypervisors and Engine Setup -**
+
+oVirt machines on versions 4.1.1 and above include Fluentd and Collectd packages.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: change to fluentd and collectd (lower case f and c)

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 26]
```diff
@@ -23,20 +23,81 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+Now we need to deploy and configure Collectd and Fluentd to send the data to the central Metrics Store:
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Change to:
fluentd and collectd (lower case f and c)

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 27]
```diff
@@ -23,20 +23,81 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+
+5. Update the file /etc/ovirt-engine-metrics/config.yml located on the engine machine, with the following lines:
+
+        # vi /etc/ovirt-engine-metrics/config.yml.example
+
+    Replace the example values with your specific environment details:
+     
+     * "fluentd-server.example.com" - The fully qualified domain name of the metrics store machine
+     
+     * "my_shared_key" - The shared key configured in Fluentd on the metrics store machine
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: change to::
fluentd (lower case f)

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 28]
```diff
@@ -23,20 +23,81 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+     
+     * "/path/to/fluentd_ca_cert.pem" - The path to the Fluentd CA certificate
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: change to fluentd

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 29]
```diff
@@ -23,20 +23,81 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+     * "ovirt_env_name" - The Environment name. Can be used to identify data collected in a single central store sent from more than one oVirt engine.
+
+6. On the engine machine, run as root:
+
+        # /usr/share/ovirt-engine-metrics/setup/ansible/configure_ovirt_hosts_for_metrics.sh
+
+It runs the Ansible script that configures Collectd and Fluentd on the oVirt engine and hypervisors.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Change to collectd and fluentd (lower case c and f)

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 25 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 26 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 30]
```diff
@@ -23,20 +23,81 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
...
+**oVirt Hypervisors and Engine Setup -**
+
+oVirt machines on versions 4.1.1 and above include Fluentd and Collectd packages.
+Now we need to deploy and configure Collectd and Fluentd to send the data to the central Metrics Store:
+
+1. Install / Upgrade and setup oVirt Engine 4.1, latest release.
+
+2. Install / Upgrade and activate one or more hosts, 4.1 latest release.
+
+3. Copy the CA certificate created earlier on [the metrics store machine](https://github.com/ViaQ/Main/blob/master/README-mux.md#getting-the-shared_key-and-ca-cert), to the engine machine.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 28 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 29 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done.

File: source/develop/release-management/features/engine/metrics-store.html.md (hunk 24 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Done

**Pull request merged**